src/Images.jsx

```diff
@@ -60,14 +60,31 @@ class Images extends React.Component {
                 });
     }
 
+    getUsedByText(image) {
+        const { imageContainerList } = this.props;
+        if (imageContainerList === null) {
+            return { title: _("unused"), count: 0 };
+        }
+        const containers = imageContainerList[image.Id + image.isSystem.toString()];
+        if (containers !== undefined) {
+            const fmt = containers.length === 1 ? _("$0 container") : _("$0 containers");
```

Comment:
Better done like this:
const fmt = cockpit.format(cockpit.ngettext("$0 container", "$0 containers", containers.length), containers.length)

Reply:
Oh nice! I should have checked https://cockpit-project.org/guide/latest/cockpit-locale.html I guess!